CMakeLists.txt

```diff
@@ -23,6 +23,8 @@ else()
     project("mbed TLS" C)
 endif()
 
+# Add CMake modules to the path
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/")
```

Comment:
Is there a well-established convention that the subdirectory has to be called `cmake`? Normally we'd put such scripts under `scripts`, either `scripts/PkgBuildConfig.cmake` or `scripts/cmake/PkgBuildConfig.cmake`.